lib/node_modules/@stdlib/blas/ext/base/sdsnansum/benchmark/benchmark.js

```diff
@@ -39,37 +44,28 @@ var sdsnansum = require( './../lib/sdsnansum.js' );
 * @returns {Function} benchmark function
 */
 function createBenchmark( len ) {
-	var x;
-	var i;
-
-	x = new Float32Array( len );
-	for ( i = 0; i < x.length; i++ ) {
-		if ( randu() < 0.2 ) {
-			x[ i ] = NaN;
-		} else {
-			x[ i ] = ( randu()*10.0 ) - 20.0;
-		}
-	}
-	return benchmark;
-
-	function benchmark( b ) {
-		var v;
-		var i;
-
-		b.tic();
-		for ( i = 0; i < b.iterations; i++ ) {
-			v = sdsnansum( x.length, x, 1 );
-			if ( isnan( v ) ) {
-				b.fail( 'should not return NaN' );
-			}
-		}
-		b.toc();
-		if ( isnan( v ) ) {
-			b.fail( 'should not return NaN' );
-		}
-		b.pass( 'benchmark finished' );
-		b.end();
-	}
+  var i = filledarrayBy( len, 'float32', rand );
+  var x = i.map(el => uniform( 0.0, 1.0 ) < 0.2 ? el : NaN)
```

Comment:
Same comment as above. Also, we use TAB indentation in JS files here and everywhere else in this PR.

Reply:
Have changed